Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerRoleStore.cs

```diff
@@ -34,6 +34,12 @@ public SqlServerRoleStore(IAuthorizationDbContext authorizationDbContext, IPermi
         }
 
         public async Task<Role> Get(Guid id)
+        {
+            var role = await GetEntityModel(id);
+            return role.ToModel();
+        }
+
+        public async Task<EntityModels.Role> GetEntityModel(Guid id)
```

Comment:
should this method be public?

Reply:
No, this returns the entity model. I'm using it internally only so that I don't have this same code repeated in other places internally.